core/src/display_object/avm1_button.rs

```diff
@@ -575,6 +579,15 @@ impl<'gc> TInteractiveObject<'gc> for Avm1Button<'gc> {
         None
     }
 
+    fn mouse_pick_avm2(
+        &self,
+        _context: &mut UpdateContext<'_, 'gc>,
+        _point: Point<Twips>,
+        _require_button_mode: bool,
+    ) -> Avm2MousePick<'gc> {
+        Avm2MousePick::NotAvm2
```

Comment:
Is this a stub-implementation? If so, might adding a `TODO` or `FIXME` comment be useful?

Reply:
No, this is a correct implementation designed to return "This button was not AVM2" instead of "Missed".